### DISCUSSION TITLE
AI for a git project

### DISCUSSION THREAD

**Discussion by XJY00** [community/community#146988] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is it feasible to support configuring an AI assistant for our individual Git repository to help us understand projects we have not encountered before? Will GitHub consider such a feature

**Comment by Logophoman** [comment]
So you're basically looking for an ai assistant for documentations? I think I've seen some GitHub repositories where you can do just that by entering the url.. you'll need a LLM and a bit of compute usually. I think you can use copilot and ask it questions about code that is public on GitHub
